verifier/client.go

```diff
@@ -5,6 +5,7 @@ package verifier
 import (
 	"context"
 
+	models "github.com/google/go-tpm-tools/internal/models"
```

Comment:
You don’t need to bind to “models” when the package basename is “models”.

Reply:
You're right, I was importing something else first in an earlier revision. Fixed.

Reply:
I thought you were right and made the change but the package is ofc "internal", so this wont work. WDYT, should I make a new package called "models" instead of co-opting this one?

Reply:
I made the package